scalalib/src/ScalaModule.scala

```diff
@@ -38,12 +38,11 @@ trait ScalaModule extends JavaModule { outer =>
    * All individual source files fed into the Zinc compiler.
    */
   override def allSourceFiles = T{
-    def isHiddenFile(path: os.Path) = path.last.startsWith(".")
     for {
       root <- allSources()
       if os.exists(root.path)
-      path <- (if (os.isDir(root.path)) os.walk(root.path) else Seq(root.path))
-      if os.isFile(path) && ((path.ext == "scala" || path.ext == "java") && !isHiddenFile(path))
+      path <- if (os.isDir(root.path)) os.walk(root.path) else Seq(root.path)
+      if os.isFile(path) && ((path.ext == "scala" || path.ext == "java") && !path.toIO.isHidden)
```

Comment:
Why?

Reply:
The `isHidden` is already defined in Java IO and is probably more robust wrt cross platform (i.e on windows a hidden file don't start with a `.`)

Reply:
Reverting